### DISCUSSION TITLE
Is it possible to add Weibull CDF & PDF as additional adstock shape functions?

### DISCUSSION THREAD

**Discussion by thipokKub** [google/lightweight_mmm#190] [Ideas]
In [Robyn](https://facebookexperimental.github.io/Robyn/docs/features/) there are `3` adstock shape function supported which are geometric, Weibull CDF, and Weibull PDF. For Lightweight MMM it support geometric, and carryover effect. Looking at the [doc](https://lightweight-mmm.readthedocs.io/en/latest/models.html), the carryover equation, it can produce S-like curve shape

![Carryover Plot](https://user-images.githubusercontent.com/21255489/227653489-d756e664-0e4b-4f8f-bfb6-868aaa6f9752.png)

which is analogous to *Weibull CDF* shape in Robyn. But *Weibull PDF* shape is missing

![Weibull Shape Robyn](https://facebookexperimental.github.io/Robyn/img/weibulladstocks.png)

So is is possible to add Weibull CDF, and PDF as additional adstock shape functions?

**Comment by statm3n** [comment]
Thank you for the feedback! We will keep monitoring the needs for Weibull for now. In the meanwhile, feel free to provide your use case where you see Weibull is a good fit.
